src/clusterfuzz/_internal/build_management/build_manager.py

```diff
@@ -550,6 +561,14 @@ def _unpack_build(self,
       utils.write_data_to_file('', partial_build_file_path)
 
     elapsed_time = time.time() - start_time
+    monitoring_metrics.JOB_BUILD_RETRIEVAL_TIME.add(
```

Comment:
It seems that the unpack time here will sometimes also include the download time since we call `_open_build_archive` on line 518, which can lead to a download.

 This is not the same behaviour as for the Custom Builds